docker/pg_tests/scripts/tests/delete_before_time_find_full_test.sh

```diff
@@ -21,34 +21,42 @@ echo "archive_timeout = 600" >> /var/lib/postgresql/10/main/postgresql.conf
 
 wal-g --config=${TMP_CONFIG} delete everything FORCE --confirm
 
-for i in 1 2 3 4
```

Comment:
I think there is no need to alter the existing tests. We should not break the current `wal-g delete` behavior for users, and these tests should prove it. Create a separate test(s) for failover scenarios.

Reply:
Took all tests back, and made only some of them check the multi-storage functionality.